argo-cd-apps/overlays/production/kustomization.yaml

```diff
@@ -73,3 +73,8 @@ patches:
       kind: ApplicationSet
       version: v1alpha1
       name: image-controller
+  - path: development-overlay-patch.yaml
```

Comment:
should be `production-overlay-patch.yaml`

Reply:
fixed